src/subscription_manager/cli_command/cli.py

```diff
@@ -239,13 +239,15 @@ def main(self, args=None):
         if args is None:
             args = sys.argv[2:]
 
-        (self.options, self.args) = self.parser.parse_known_args(args)
+        (self.options, unknown_args) = self.parser.parse_known_args(args)
 
         # check for unparsed arguments
-        if self.args:
-            for arg in self.args:
-                print(_("cannot parse argument: {}").format(arg))
-            system_exit(os.EX_USAGE)
+        if unknown_args:
+            message: str = _("{prog}: no such option: {args}").format(
```

Comment:
can you please add "error: " between the program name and the message? this way it is more clear it is an error

Reply:
That's a lot of colons for one message :)

Is the program name neccesary? Looking at the listing below it seems that it's too much.
```bash
error: no such option: --foo bar
subscription-manager: no such option: --foo bar
subscription-manager: error: no such option: --foo bar
```